### ISSUE TITLE
Deleted notes keep reappearing

### ISSUE THREAD

**Issue by Poecilia** [standardnotes/forum#353]
I found out about Standard Notes via the [/e/ beta testing community](https://community.e.foundation/) and I really like it so far. However there is this issue:

I exported all my Evernote notes with tags and had them stripped of html by the online tool, then imported all of them (thousands). I have Standard Notes on my Macbook air, on a Thinkpad x220 with Linux and on my phone (with /e/ Android fork in beta testing - here I downloaded Standard Notes from the Yalp store). 

The Linux version looks good: all notes and all tags are gone. The phone and Mac versions don't: all the notes keep showing up, even after uninstalling Standard Notes and then reinstalling.

I'd really like to use Standard Notes and am considering going premium, but I need to understand what I can do to sort this issue.

**Comment by Poecilia** [comment]
Update: actually, deleted notes were coming back on all three devices. What I did was delete the app from my phone, sign out of Standard Notes on the Mac. I then deleted all notes and tags on the Linux device. I then found the "Standard Notes Backups" folder on both the Linux laptop and the Mac and deleted all the txt files there. This seems to have worked, and individual deleted test notes are synced as deleted across all three devices.

However, I should not need to delete my backup files after batch deleting a lot of notes in order to avoid them coming back from the dead. That is a confusing process.

**Comment by mobitar** [comment]
Hey @Poecilia, might be some confusion here. You do not have to delete the backup files to affect syncing at all. Those have no effect and are just isolated backups. Also, I'm not familiar with any releases on the Yalp store. Those are unofficial as far as I'm concerned. If they are community maintained, you have to check to see you have the latest version (2.3.7 for Android). Can you confirm this is the version you are running?

Lastly, it's possible that complications may have arisen if after importing the few thousand items, you did not allow syncing to complete. The best way to tackle this in the future if you see any issues importing thousands of items is to just sign out and sign back in on any affected devices. 

For desktop, uninstalling the app won't always be enough, especially on Mac, where deleting the .app file does not delete your data directory. So you'd have to manually sign out and sign back in to get at the freshest state.

**Comment by Poecilia** [comment]
Hi Mobitar,

Thanks for the reply. The version on my phone is 2.3.7.

To respond to your point - I had Standard Notes up and running for several days, so syncing was completed. The problem was that whenever I tried to use batch extension to bulk delete notes, it would be empty for a short while, and then the same notes would reappear (I recognised them). I tried everything, including signing out and in, uninstalling, restarting all devices, etc. When I found and deleted the backup files in the Standard Notes Backups folders on my devices, the problem of notes reappearing stopped immediately, so I can't help but conclude that that's what did the trick in the end.
